crates/config/src/fuzz.rs

```diff
@@ -22,6 +26,10 @@ pub struct FuzzConfig {
     /// The fuzz dictionary configuration
     #[serde(flatten)]
     pub dictionary: FuzzDictionaryConfig,
+    /// Path where fuzz failures are recorded and replayed, defaults to `~/.foundry/cache/fuzz`
```

Comment:
Should we persist the data in the project's local `./cache` folder instead? I haven't looked at exactly what's in the file, but if the fuzz data is unique to the project we probably should use the local cache. cc @klkvr

Reply:
Yeah, this should definitely be in somewhere in project's root. Let's write to `./{cache_path}/fuzz` by default.

Reply:
> Should we persist the data in the project's local `./cache` folder instead? I haven't looked at exactly what's in the file, but if the fuzz data is unique to the project we probably should use the local cache. cc @klkvr

yeah, the fuzz data is unique per fuzzed test